docs/architecture/adr-055-protobuf-design.md

```diff
@@ -0,0 +1,45 @@
+# ADR 055: Protobuf Design
+
+## Changelog
+
+- 2020-03-31: Created
```

Comment:
do you want to put your name on this? that seems to be the convention. 

(otoh, not sure why every ADR needs a changelog when everything gets checked into git anyways.)